cmd/ci-secret-generator/main.go

```diff
@@ -170,16 +249,27 @@ func updateSecrets(bwItems []bitWardenItem, bwClient bitwarden.Client) error {
 			logger.Info("processing attribute")
 			out, err := executeCommand(bwItem.Attribute.Cmd)
 			if err != nil {
-				logrus.WithError(err).Error("failed to generate attribute")
-				return err
+				errs = append(errs, fmt.Errorf("bwItem.ItemName: %s, bwItem.Password:, %s failed: %w", bwItem.ItemName, bwItem.Attribute.Cmd, err))
```

Comment:
Why are we getting back to this error message? It's not useful, the logged and structured one is

Reply:
I am also outputting the earlier structure of the error message on the line below, but I am also returning the wrapped error message